src/main/java/org/elasticsearch/indices/InternalIndicesLifecycle.java

```diff
@@ -245,4 +245,15 @@ public void indexShardStateChanged(IndexShard indexShard, @Nullable IndexShardSt
             }
         }
     }
+
```

Comment:
It does not override any method from the superclass. onShardInactive() is a method in IndicesLifecycle.Listener.